app/models/spree/order_decorator.rb

```diff
@@ -17,6 +17,19 @@ def cancel_avalara
     self.avalara_transaction.cancel_order
   end
 
+  def adjust_avalara
+    logger.debug 'adjust avalara order'
+    begin
+      create_avalara_transaction if avalara_transaction.nil?
+      @rtn_tax = self.avalara_transaction.adjust_avatax
+      logger.info_and_debug('tax amount', @rtn_tax)
```

Comment:
`info_and_debug` is very weird. The `info` logger level automatically covers `debug`, as they are:
1. debug
2. info
3. warning
4. error